### PR TITLE
Minimal documentation of minimal component set for PRL sim

### PR DESCRIPTION
**Pull request by aflaxman**

### CODE REVIEW

File: docs/source/concept_models/vivarium_census_synthdata/concept_model.rst [hunk 1]
```diff
@@ -479,20 +479,272 @@ When we reach that point, we might also want to think about the change
...
+    for (age,sex), df_age in df_population.groupby(['age', 'sex']):
+        df_population.loc[df_age.index, 'first_name'] = random_names(age, sex, len(df_age))
+        df_population.loc[df_age.index, 'middle_name'] = random_names(age, sex, len(df_age))
+
+
+2.3.7 Component 11: Date of Birth
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To create a date-of-birth column in the synthetic output data, each
+simulant should have a uniformly random date of birth which is
```
Comment (alibow) [inline docs/source/concept_models/vivarium_census_synthdata/concept_model.rst:643]: Would it be worth using an actual distribution of birthdays throughout the year? I think there are slight seasonal differences in density!

**Review by alibow — APPROVED** [review]

File: docs/source/concept_models/vivarium_census_synthdata/concept_model.rst [hunk 2]
```diff
@@ -479,20 +479,272 @@ When we reach that point, we might also want to think about the change
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Eventually, this should be missing for some and not actually unique
+for others.  I need to do some research into how we represent this,
+and how important it is.
+
+There is a python library that includes a detailed SSN generation
+module, but it doesn't have a reference, so I would also like to do some
+research to check if it is right:
+https://github.com/joke2k/faker/blob/master/faker/providers/ssn/en_US/__init__.py#L219-L222
```
[the remark this answers is not in the document]
Comment (aflaxman) [reply docs/source/concept_models/vivarium_census_synthdata/concept_model.rst:666]: 👍

File: docs/source/concept_models/vivarium_census_synthdata/concept_model.rst [hunk 3]
```diff
@@ -479,20 +479,272 @@ When we reach that point, we might also want to think about the change
...
+point down the road).
+
+Adding noise to the fields in these observers will be another
+important part of the art, but this can happen _after_ simulation.
+Some existing projects with noisy include
+https://github.com/pinformatics/rlErrorGeneratoR and GeCo.
+
+GeCO distinguishes keyboard, transcription, and OCR error, and despite
+being unsupported for 10 years, it seems to be the standard approach
+among methods researchers, so we might aim for replicating it. The
```
[the remark this answers is not in the document]
Comment (aflaxman) [reply docs/source/concept_models/vivarium_census_synthdata/concept_model.rst:711]: All good questions, to be investigated further as we obtain access to the data that is actually being linked.

File: docs/source/concept_models/vivarium_census_synthdata/concept_model.rst [hunk 4]
```diff
@@ -479,20 +479,272 @@ When we reach that point, we might also want to think about the change
...
 
-2.3.5 Component 9+
-~~~~~~~~~~~~~~~~~~
-To Come (TK)
+2.3.5 Component 9: Address
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each household id should be associated with a postal address, and when
+people move, they should often move into previously vacated
+households, so that there are collisions between residential
```
[the remark this answers is not in the document]
Comment (alibow) [reply docs/source/concept_models/vivarium_census_synthdata/concept_model.rst]: Yes, thanks!

**Pull request merged**